src/Altinn.App.Api/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -54,7 +57,22 @@ public static void AddAltinnAppServices(this IServiceCollection services, IConfi
             services.AddAppServices(config, env);
             services.ConfigureDataProtection();
 
-            AddApplicationInsights(services, config, env);
+            var useOpenTelemetrySetting = config["AppSettings:UseOpenTelemetry"];
```

Comment:
Doesn't this perform much of the same function?
```suggestion
            var useOpenTelemetrySetting = config.GetValue<bool>("AppSettings:UseOpenTelemetry");
```